scene/2d/camera_2d.cpp

```diff
@@ -281,11 +281,10 @@ void Camera2D::_notification(int p_what) {
 				inv_camera_transform.xform(Vector2(0, screen_size.height))
 			};
 
-			Matrix32 inv_transform = get_transform().affine_inverse(); // undo global space
-			draw_set_transform(inv_transform.get_origin(), inv_transform.get_rotation(), inv_transform.get_scale());
```

Comment:
When the parent was scaled unequally, `draw_set_transform` failed me, so I had to switch :smile: